examples/studio/assistant/assistant.py

```
@@ -0,0 +1,39 @@
+from ai21 import AI21Client
+
+
+def main():
+    ai21_client = AI21Client()
+
+    assistant = ai21_client.beta.assistants.create(name="My Assistant")
+
+    thread = ai21_client.beta.threads.create(
+        messages=[
+            {
+                "role": "user",
+                "content": {
+                    "type": "text",
+                    "text": "Hello",
+                },
+            },
+        ]
+    )
+
+    run = ai21_client.beta.threads.runs.create(
+        thread_id=thread.id,
+        assistant_id=assistant.id,
+    )
+
+    while run.status == "in_progress":
+        run = ai21_client.beta.threads.runs.retrieve(thread_id=thread.id, run_id=run.id)
+
+    if run.status == "completed":
+        messages = ai21_client.beta.threads.messages.list(thread_id=thread.id)
+        print("Messages:")
+        print("\n".join(f"{msg.role}: {msg.content['text']}" for msg in messages.results))
+    else:
+        # handle error or required action
+        pass
```

Lets raise an error here so our test would fail on it